internal/pkg/cli/env_init.go

```diff
@@ -329,12 +329,11 @@ func (o *initEnvOpts) validateCustomizedResources() error {
 		return fmt.Errorf("cannot import or configure vpc if --%s is set", defaultConfigFlag)
 	}
 	if o.importVPC.isSet() {
-		// We allow 0 or 2+ public subnets.
+		// Allow passing in VPC without subnets, but error out early for too few subnets-- we won't prompt the user to select more of one type if they pass in any.
 		if len(o.importVPC.PublicSubnetIDs) == 1 {
-			return fmt.Errorf("at least two public subnets must be imported to enable Load Balancing")
+			log.Warningf("At least two public subnets must be imported to enable Load Balancing.\n")
```

Comment:
Should we still return an error here, but instead we can check if they set the flag and if they did then they must have specified at least two.

```go
if n := o.importVPC.PublicSubnets; n > 0 && n < 2 {
   return errors.New("at least two public subnets must be imported to enable Load Balancing")
}
```

Reply:
But shouldn't they be allowed to import 1 public and 2 private if they don't want load balancing?

Reply:
Hmm that's a good question!

My intuition was that we should be consistent with `Ask` (where we err if only 1 public subnet is selected)
I think for this scenario, we can err on the side of being restrictive at first and then relax it if customers ask for it.